diff -Nur pypolicyd-spf-0.5.2~rc1/policyd-spf pypolicyd-spf-0.5.2~rc1.new/policyd-spf
--- pypolicyd-spf-0.5.2~rc1/policyd-spf	2007-10-25 17:04:52.000000000 -0400
+++ pypolicyd-spf-0.5.2~rc1.new/policyd-spf	2007-10-25 17:47:50.000000000 -0400
@@ -25,7 +25,7 @@
 __version__ = "0.5.2: October 25, 2007"
 
 import syslog, os, sys, string, re, time, popen2, urllib, stat, errno, socket, spf
-sys.path.append('/usr/local/lib/policy-spf')
+
 import policydspfsupp
 
 syslog.openlog(os.path.basename(sys.argv[0]), syslog.LOG_PID, syslog.LOG_MAIL)
